vmck/base_settings.py

```diff
@@ -53,7 +55,7 @@
 STATIC_URL = '/static/'
 
 NOMAD_JOB_PREFIX = 'vmck-'
-NOMAD_DEPLOYMENT_NAME = "vmck"
+NOMAD_DEPLOYMENT_NAME = f"{gethostname()} vmck/{os.environ.get('DRONE_BRANCH', 'master')}"  # noqa: E501
```

Comment:
the `DRONE_BRANCH` environ that we might want to display is only available in the build, not in the vmck server, so there's no point reading it here.

Reply:
I presume we drop it for good then. What should I use instead?

Reply:
How about `vmck@{host}#{job.id} {job.name}`?
* `hostname` is configurable through settings and env var, defaults to `gethostname()`.
* `job.name` comes from the client, you need to save that name on the Job model.

Reply:
![test](https://user-images.githubusercontent.com/24795454/62838855-60f9c080-bc8a-11e9-80d8-7c0ef53f15d5.png)
Got the new format! I'll add an option to vagrant-vmck to set the name too, until then we will only have `UNSET` values.

Reply:
You can use `"default"` as the default, since that is the default machine name for Vagrant, so it will become the default in people's default vagrantfiles if they don't change the default.

Reply:
It's done now! @mgax